src/core_atmosphere/physics/mpas_atmphys_driver_radiation_lw.F

```diff
@@ -402,8 +402,14 @@ subroutine radiation_lw_from_MPAS(xtime_s,configs,mesh,state,time_lev,diag_physi
  call mpas_pool_get_field(diag_physics,'m_hybi'    ,m_hybiField )
  call mpas_pool_get_field(diag_physics,'m_ps'      ,m_psField,time_lev)
 
- if (mpas_cpl%role_is(ROLE_INTEGRATE) .or. &
-     mpas_cpl%role_is(ROLE_RADIATION)) then
+ if (mpas_cpl%role_includes(ROLE_INTEGRATE)) then
+    !$acc update host(latCell, lonCell, aerosols, skintemp, snow, xice, &
+    !$acc xland, pin, ozmixm, sfc_albedo, sfc_emiss, cldfrac, o3clim, &
+    !$acc o3vmr, m_hybi, m_ps)
+ end if
+
+ if (mpas_cpl%role_includes(ROLE_INTEGRATE) .or. &
+     mpas_cpl%role_includes(ROLE_RADIATION)) then
```

Comment:
The original logic here was:
```fortran
 if (mpas_cpl%role_is(ROLE_INTEGRATE) .or. &
     mpas_cpl%role_is(ROLE_RADIATION)) then
```
which ensured that fields were transferred only if MPI tasks were performing an exclusive role (either INTEGRATE or RADIATION). The new logic,
```fortran
 if (mpas_cpl%role_includes(ROLE_INTEGRATE) .or. &
     mpas_cpl%role_includes(ROLE_RADIATION)) then
```
will transfer fields even in cases where MPI tasks are performing both INTEGRATE and RADIATION roles. Is this the intent?